src/collections/hash/map.rs

```diff
@@ -2054,17 +2054,15 @@ impl RandomState {
         // iteration order allows a form of DOS attack. To counter that we
         // increment one of the seeds on every RandomState creation, giving
         // every corresponding HashMap a different iteration order.
-        thread_local!(static KEYS: Cell<(u64, u64)> = {
```

Comment:
Also, could you add a note here and the tracking issue that we still have to move this to use TLS?